ibis/pyspark/client.py

```diff
@@ -19,30 +21,39 @@ def __init__(self, session):
         super().__init__(session)
         self.translator = PySparkExprTranslator()
 
-    def compile(self, expr, params=None, *args, **kwargs):
+    def compile(self, expr, timecontext=None, params=None, *args, **kwargs):
         """Compile an ibis expression to a PySpark DataFrame object
         """
 
         # Insert params in scope
         if params is None:
-            scope = {}
+            scope = Scope()
         else:
-            scope = dict(
-                (param.op(), raw_value) for param, raw_value in params.items()
+            scope = Scope().merge_scopes(
```

Comment:
I think ideally there should be a way to create a scope that with n items without having to create nn scope with one item and then merge them.

Can you add implement a constructor that does this?